internal/client.go

```diff
@@ -213,61 +224,113 @@ func (client *Client) setupUI() {
 	client.app.EnableMouse(true)
 }
 
-func (client *Client) shortenPlotId(id string) string {
+func shortenPlotId(id string) string {
```

Comment:
I may end up putting this back in to `Client` at some point, it would be nice if this dynamically adjusted to the screen size, but I haven't looked in to the difficulty of doing that.  For now at least, it's not required.